### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by Lenmiuuu**
Created Splash Screen for the Game

### CODE REVIEW

**Comment by Lenmiuuu** [comment]
Missing the Main Screen when the Splash Screen is Finished.

File: src/main/java/com/coding4buddies/batshooter/SplashScreen.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+       });
+    }
+    
+    // Method that insert the progress bar and label
+    public void addLoading(){
+        label.setAlignmentX(Component.CENTER_ALIGNMENT);   
+        progressBar.setMaximumSize(new Dimension(400,10));
+        this.setLayout(new BoxLayout(this.getContentPane(), BoxLayout.Y_AXIS));
+        this.add(Box.createVerticalStrut(400));
+        this.add(label);
```
Comment (Jookie262) [inline src/main/java/com/coding4buddies/batshooter/SplashScreen.java:59]: Make the color of JLabel to black and adjust the font size. Also add a space between the progress bar.

**Review by Jookie262 — CHANGES_REQUESTED** [review]
Apply changes in the Label area

File: src/main/java/com/coding4buddies/batshooter/SplashScreen.java (hunk 1 above)
Comment (Jookie262) [reply src/main/java/com/coding4buddies/batshooter/SplashScreen.java:59]: Also if possible, change the image of splash screen

**Review by Jookie262 — APPROVED** [review]
Everything is good!

**Pull request merged**